build/tasks/task-serve.js

```diff
@@ -0,0 +1,34 @@
+// Any copyright is dedicated to the Public Domain.
+// http://creativecommons.org/publicdomain/zero/1.0/
+
+import gulp from 'gulp';
+import portastic from 'portastic';
+
+import logger from '../logger';
+
+import { spawn } from '../../src/shared/util/spawn';
+import * as Paths from '../../src/shared/paths';
+
+const PORT_RANGE = {
+  min: 9000,
+  max: 9999,
+};
+
+gulp.task('serve', async () => {
```

Comment:
I think we should have a better name for this task, but I don't know what would be better here. Certainly "serve" implies that only a server starts, but this does a tiny bit more. Otherwise the `start` task might want to use the `serve` task, if the latter was thinner. I think this is fine for now, but in the interest of making tasks more modular and only do one thing, we should think this through a bit.